docs/install.md

```diff
@@ -559,10 +559,13 @@ Select the following tabs for the installation description of your target.
 
 ## 4. First logon on DietPi
 
+The hostname of the system will be: **dietpi**.  
```

Comment:
It will be "DietPi". Hostnames are case-sensitive.

Reply:
Thx. Changed.